changelog/1.28.4.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "1.28.4"
+description: "Released on 03/17/2022"
+---
+
+### Breaking changes ❗
+
+There are no breaking changes in 1.28.4.
+
+### Features ✨
+
+- web: added hosting for the **Open in Coder SVGs** in Coder so that the
+  icons render properly in firewalled networks. Coder will continue to support
+  the existing external CDN for such icons, but the default will be the SVG
+  hosted by Coder.
```

Comment:
The wording for this seems confusing, how about:

> web: the **Open in Coder** SVG is now served from your Coder deployment rather than our CDN so airgapped customers can use this feature. Existing button images will continue to be served from our CDN until updated.